lib/viewcomponents/app.jsx

```diff
@@ -10,9 +10,9 @@ var App = React.createClass({
         return (
             <div id='app'>
                 <Graph model={this.props.model} />
-                <div id='model_editor'>
-                    <textarea onChange={this._onModelChange} value={JSON.stringify(this.props.model.getContents(), null, '  ')} />
-                </div>
+                {/*<div id='model_editor'>
+                    //<textarea onChange={this._onModelChange} value={JSON.stringify(this.props.model.getContents(), null, '  ')} />
+                //</div>*/}
```

Comment:
Damn. Okay. I hope we get the parser working soon, so that we can add back the editor panel, but with GCL in place of JSON.